### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by egemenkorkmaz** [mckaywrigley/chatbot-ui#1303] [Help]
I am creating a membership but this error is always present

![Ekran Resmi 2024-01-29 00 41 41](https://github.com/mckaywrigley/chatbot-ui/assets/19240650/e5335455-c6bc-4c48-8ce1-b128ff1b1952)
![Ekran Resmi 2024-01-29 00 42 39](https://github.com/mckaywrigley/chatbot-ui/assets/19240650/b9855c6c-1f48-4211-85bf-eb64ab993d83)

**Comment by solaius** [comment]
This happened after I restarted my computer. I had to restart docker and supabase and it started working again.
